pkg/kubelet/eviction/helpers.go

```diff
@@ -1243,6 +1252,23 @@ func evictionMessage(resourceToReclaim v1.ResourceName, pod *v1.Pod, stats stats
 	if !ok {
 		return
 	}
+
+	// Pod level resources will be included in eviction message, along container resources
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources) && resourcehelper.IsPodLevelResourcesSet(pod) && resourcehelper.IsSupportedPodLevelResource(resourceToReclaim) {
+		podRequest := v1resource.GetResourceRequestQuantity(pod, resourceToReclaim)
```

Comment:
Per-container usage is checked below. Does it still make sense to aggregate the container-level requests here if the pod-level requests aren't set?